core/src/style/polylineStyle.cpp

```diff
@@ -128,6 +130,13 @@ void PolylineStyle::constructShaderProgram() {
     if (m_dashArray.size() > 0 || m_texture) {
         m_shaderProgram->addSourceBlock("defines", "#define TANGRAM_LINE_TEXTURE\n", false);
         m_shaderProgram->addSourceBlock("defines", "#define TANGRAM_ALPHA_TEST 0.25\n", false);
+        if (m_dashArray.size() > 0) {
+            m_shaderProgram->addSourceBlock("defines", "#define TANGRAM_DASHLINE_TEX_SCALE " +
+                                                std::to_string(dash_scale) + "\n", false);
+        } else {
+            m_shaderProgram->addSourceBlock("defines", "#define TANGRAM_DASHLINE_TEX_SCALE 1",
```

Comment:
Have you tested this case? The way this is used in the shader I think this could cause a float-int mismatch error.